lib/alpha_taxonomy/taxon_presenter.rb

```diff
@@ -0,0 +1,41 @@
+module AlphaTaxonomy
+  class TaxonPresenter
+    attr_accessor :title, :slug
+
+    def initialize(title:)
+      @title = title
+      validate_title
+      @slug  = title.parameterize
+    end
+
+    def present
+      {
+        base_path: base_path,
+        format: "taxon",
+        title: @title,
+        publishing_app: 'content-tagger',
+        rendering_app: 'collections',
+        public_updated_at: DateTime.current.to_s,
```

Comment:
Given the wide range of date formats, I think being explicit is probably better (ie, would prefer to see `iso_8601` used)